leetcode/src/236.c

```diff
@@ -0,0 +1,80 @@
+/**
+ * Definition for a binary tree node.
+ * struct TreeNode {
+ *     int val;
+ *     struct TreeNode *left;
+ *     struct TreeNode *right;
+ * };
+ */
+
+struct List {
+    struct TreeNode* node;
+    struct List* next;

```

Comment:
Could you add a comment explaining why you added this? I believe you don't need an array of `List` for this usually (unless I'm mistaken) so documenting why you added it is better for learning 😄

Reply:
Yes, sure)) probably it should be `ListItem`. Sorry)